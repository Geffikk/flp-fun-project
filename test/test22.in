S
a,b,c
S
S->aSa
S->bSb
S->c
