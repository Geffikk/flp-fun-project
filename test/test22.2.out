S,a',<Sa>,<Sb>,b'
a,b,c
S
S->a'<Sa>
a'->a
<Sa>->Sa'
S->b'<Sb>
b'->b
<Sb>->Sb'
S->c
